ui/revenuecatui/src/test/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/VariableDataProviderTest.kt

```diff
@@ -1,50 +0,0 @@
-package com.revenuecat.purchases.ui.revenuecatui.data.processed
-
-import androidx.test.ext.junit.runners.AndroidJUnit4
-import com.revenuecat.purchases.ui.revenuecatui.data.TestData
-import com.revenuecat.purchases.ui.revenuecatui.helpers.ApplicationContext
-import io.mockk.every
-import io.mockk.mockk
-import org.assertj.core.api.Assertions.assertThat
-import org.junit.Before
-import org.junit.Test
-import org.junit.runner.RunWith
-import java.util.Locale
-
-@RunWith(AndroidJUnit4::class)
-class VariableDataProviderTest {
```

Comment:
I removed these tests since most of this is covered in `VariableProcessorTest` so it seemed redundant. Lmk if you think we should keep it.